package.json

```diff
@@ -137,23 +138,30 @@
   },
   "homepage": "https://threejs.org/",
   "devDependencies": {
-    "@babel/core": "^7.18.2",
+    "@babel/core": "^7.18.5",
     "@babel/eslint-parser": "^7.18.2",
     "@babel/plugin-proposal-class-properties": "^7.17.12",
     "@babel/preset-env": "^7.18.2",
     "@rollup/plugin-babel": "^5.3.1",
     "@rollup/plugin-node-resolve": "^13.3.0",
     "chalk": "^5.0.1",
-    "concurrently": "^7.2.1",
-    "eslint": "^8.16.0",
+    "concurrently": "^7.2.2",
+    "eslint": "^8.17.0",
     "eslint-config-mdcs": "^5.0.0",
     "eslint-plugin-compat": "^4.0.2",
     "eslint-plugin-html": "^6.2.0",
     "eslint-plugin-import": "^2.26.0",
-    "rollup": "^2.75.0",
+    "failonlyreporter": "^1.0.0",
+    "jimp": "^0.16.1",
+    "node-fetch": "^3.2.6",
+    "pixelmatch": "^5.2.1",
+    "puppeteer-core": "^14.4.1",
+    "qunit": "^2.18.0",
+    "rollup": "^2.75.6",
     "rollup-plugin-filesize": "^9.1.2",
     "rollup-plugin-terser": "^7.0.2",
     "rollup-plugin-visualizer": "^5.6.0",
+    "serve-handler": "^6.1.3",
```

Comment:
If you using `servez` that have `express` in dependency, you don't need `serve-handler` anymore.

Reply:
Hm, how exactly can it be replaced?

Reply:
It is used almost the [same](https://dev.to/heyshadowsmith/how-to-make-an-api-from-scraped-data-using-express-puppeteer-2n7e).

Reply:
I think express is used there only to return a string/array, not to serve a directory?

Reply:
It is possible to share a folder with it, but maybe this will work too.
```js
const Servez = require('servez-lib');

const server = new Servez({
  './examples',
  port: 8080,
  scan: true,
  extensions: ['html'],
});
server.on('start', startInfo => {
  async () => await puppeteerLaunch
});
server.on('error', msg => console.warn(msg));
```

Reply:
Found a solution with express!